ikarus/finiteelements/mechanics/linearelastic.hh

```diff
@@ -75,7 +75,7 @@ namespace Ikarus {
     LinearElastic(const Basis& globalBasis, const typename LocalView::Element& element, double emod, double nu,
                   VolumeLoad p_volumeLoad = {}, const BoundaryPatch<GridView>* p_neumannBoundary = nullptr,
                   NeumannBoundaryLoad p_neumannBoundaryLoad = {})
-        : BaseDisp(globalBasis.flat(), element), neumannBoundary{p_neumannBoundary}, emod_{emod}, nu_{nu} {
+        : BaseDisp(globalBasis.flat(), element), emod_{emod}, nu_{nu}, volumeLoad(globalBasis, element, p_volumeLoad), tractionLoad(globalBasis, element, p_neumannBoundary, p_neumannBoundaryLoad) {
```

Comment:
If you inherit from it why do yo also add it as a member variable? This is not the way to go. Look up how to initialize Bas classes.